frontend/js/components/map.js

```diff
@@ -139,15 +138,11 @@ export class LeafletMap extends React.Component {
             return <IssueMarker {...props} />
         })
 
-        let extents = geojsonExtent(JSON.parse(JSON.stringify(geojson)));
```

Comment:
the fetchIssueIfNeeded is just a stub that can and should be implemented further.
This is the point where we can decide if the issue details should be reloaded (e.g. in response to your suggested commentAdded/Deletred action)

For reasons of simplicity we currently just always load the issue. But I agree, the requestIssues action is not needed there, but it does hide another bug ;-) 
If I remove that then all markers vanish....